src/kernel/chainstatemanager_opts.h

```diff
@@ -5,11 +5,16 @@
 #ifndef BITCOIN_KERNEL_CHAINSTATEMANAGER_OPTS_H
 #define BITCOIN_KERNEL_CHAINSTATEMANAGER_OPTS_H
 
+#include <chrono>
 #include <cstdint>
 #include <functional>
 
 class CChainParams;
 
+using namespace std::chrono_literals;
```

Comment:
In commit "Move ::nMaxTipAge into ChainstateManager" (1111ff10084e4446c2fbcaff65e6f0aa2a66a83e)

Was going to suggest it would be better to avoid `using namespace` at global level in a header file, since it means anything that directly or indirectly includes the header has now has std symbols in the global namespace.

But I guess `src/qt/guiconstants.h` and `src/util/time.h` already do this so maybe we don't care.

Reply:
Jup, that was the goal of `src/util/time.h`. Happy to switch to using that include. Alternatively I can review a pull that changes this if there is need to and someone creates one.

Reply:
> Jup, that was the goal of `src/util/time.h`. Happy to switch to using that include. Alternatively I can review a pull that changes this if there is need to and someone creates one.

I'd go the opposite direction. I think it's better to say `using namespace std::chrono_literals;` than include `time.h`, which just makes it harder to understand where the literals are coming from. IMO it would be best to delete `using namespace std::chrono_literals;` from `time.h` and add it explicitly to any file that use literals.

My initial reaction was an objection to having `using namespace` in a header file at all since it is common advice "Namespace using declarations should never appear in header files" for reasons described https://stackoverflow.com/a/37376171. But these reasons do not really apply to namespaces specifically reserved for literals like chrono_literals.

Reply:
Happy to push any diff here or review a pull, if there is a motivation for the change.

Reply:
Removed the namespace (though, conceptually I did the opposite of what you asked for)